src/components/app/details/appConfig/AppConfig.tsx

```diff
@@ -280,35 +314,50 @@ const NextButton: React.FC<{ isCiPipeline: boolean; navItems, currentStageName,
     return null;
 };
 
-function Navigation({ navItems, deleteApp }) {
+function Navigation({ navItems, deleteApp, isCDPipeline}) {
+const pathname = window.location.pathname;
```

Comment:
Suggestion - Can we try using `location` from react-router instead of the window object.